examples/js/exporters/GLTFExporter.js

```diff
@@ -1119,8 +1130,20 @@ THREE.GLTFExporter.prototype = {
 
 			}
 
+			if ( primitives.length === 0 ) {
+
+				return null;
+
+			}
+
```

Comment:
Adding 

    if ( Object.keys( attributes ).length === 0 ) {

        return null;

    }

after

    for ( var attributeName in geometry.attributes ) {

        ....

    }

above instead isn't good?

Reply:
Is there any use case where you could have just morph attributes but no other attribute?

Reply:
IMO, no. At least, glTF validator seems to raise MESH_PRIMITIVE_MORPH_TARGET_NO_BASE_ACCESSOR "No base accessor for this attribute semantic." error if corresponding attribute isn't in  `attributes`.

Reply:
Yeah makes sense, added! thanks

Reply:
Nice. So I think you can remove this check here now.

Reply:
@fernandojsg ping

Reply:
@mrdoob I believe he's referring to this https://github.com/mrdoob/three.js/pull/13681#pullrequestreview-106725106 right? /cc @takahirox

Reply:
No. I think we can remove here

	if ( primitives.length === 0 ) {

		return null;

	}

because we added above

	if ( Object.keys( attributes ).length === 0 ) {

		return null;

	}

Or you care the case where `groups` is an empty array?

Reply:
You could have buffer geometries without `groups` eg: the sphere in the example https://github.com/mrdoob/three.js/blob/dev/examples/misc_exporter_gltf.html#L248 or a custom buffergeometry: https://github.com/mrdoob/three.js/blob/dev/examples/misc_exporter_gltf.html#L312-L337 but I haven't worked with multimaterial before so I'm not sure how that case would be handled here. 
Because if you have one of the previous objects with a multimaterial.
```
var groups = isMultiMaterial ? mesh.geometry.groups : [ { materialIndex: 0, start: undefined, count: undefined } ];
```
`groups` would be `mesh.geometry.groups` that would be empty.

Reply:
I confirmed rendering nothing for multi-material with empty groups.

So how about simply

    if ( isMultiMaterial && mesh.geometry.groups.length === 0 ) return null;

it'd be more straightforward.

Reply:
@takahirox thanks! done